buildlib/tools/builds.sh

```diff
@@ -461,5 +461,5 @@ do
 	[ -d "${ucx_build_dir}" ] && rm -rf ${ucx_build_dir}/*
 
 	# run the test
-	${test_name}
+	${test_name} || { echo "Test failed: ${test_name}"; exit 1; }
```

Comment:
Maybe better use `azure_log` helpers here instead, @yosefe  wdyt?

Reply:
Changed `echo` to `azure_log_error`, please check @gleon99 @yosefe